AppKit/CPPopUpButton.j

```diff
@@ -491,9 +491,17 @@ CPPopUpButtonStatePullsDown = CPThemeState("pulls-down");
 
         if (index < 0)
         {
-            [self addItemWithTitle:aTitle];
 
-            index = [self numberOfItems] - 1;
+            // this ist to match cocoa where setting an empty string does not add it but simply clears the title
+            // and sets objectValue to -1
+            if (aTitle === '')
+                [self selectItemAtIndex:-1];
```

Comment:
What about `index = -1` ?

Reply:
i am not sure i understand you question 100%.

index = -1 means "this popupbutton has currently no selected item".

Reply:
i have to check tonight the value in cocoa with bindings.

Reply:
@cacaodev the object value seems to equal the selected index in cocoa when the popupbutton is filled via bindings, regardless whether the title is the empty string.